client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -63,6 +65,28 @@ export class CartItem extends React.Component {
 			return <span className="cart__free-text">{ translate( 'Free' ) }</span>;
 		}
 
+		if ( isGSuiteProductSlug( cartItem.product_slug ) ) {
+			const { cost_before_coupon: costBeforeCoupon } = cartItem;
+
+			if ( costBeforeCoupon ) {
+				return (
+					<div className="cart__gsuite-discount">
+						<span className="cart__gsuite-discount-regular-price">
+							{ costBeforeCoupon }
+						</span>
+
+						<span className="cart__gsuite-discount-discounted-price">
+							{ cost } { cartItem.currency }
+						</span>
+
+						<span className="cart__gsuite-discount-text">
+							{ translate( 'First year discount' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Discount for first year' )` *ES Score: 9*

Reply:
I suppose that's a good option, bot. @fditrapani, let me know what you think of the copy 🙂.

Reply:
The copy looks good to me 👍